app/src/main/java/ch/sdp/vibester/activity/WelcomeActivity.kt

```diff
@@ -29,13 +29,13 @@ class WelcomeActivity : AppCompatActivity() {
     }
 
     fun switchToProfile(view: View) { //FILLER INTENT
-//        sendDirectIntent(GameSetupScreen::class.java)
```

Comment:
Same as the previous two comments, just a small reminder for next week to refactor this code here.